main.bal

```diff
@@ -334,8 +334,9 @@ service / on new http:Listener(serverPort) {
     #
     # + page - the requested page (starting with page 0)
     # + 'item\-count - the number of items per page (5 <= item-count <= 500)
+    # + ascending - 1 for asc sort, 0 for desc sort by step sequence
     # + return - the list resource containing the experiments
-    resource function get experiments(int? page = 0, int? 'item\-count = 10) returns ExperimentListResponse|http:InternalServerError|http:BadRequest|http:NotFound {
+    resource function get experiments(int? page = 0, int? 'item\-count = 10, int? 'ascending = 1) returns ExperimentListResponse|http:InternalServerError|http:BadRequest|http:NotFound {
```

Comment:
A search for experiments also seams like a good idea (can be implemented later but then please open an issue to track this).